final/discuss-recommend/eclipse.tex

```diff
@@ -0,0 +1,39 @@
+The Eclipse frontend that is delivered with the product has been discussed in
+\cref{ssec:eclipse-plugin}. The plugin provides interaction with the REPL
+backend, as discussed in \cref{chap:design}.
+
+The delivered Eclipse frontend is just that: a frontend to the REPL backend.
+Integration with Eclipse and Spoofax Eclipse are not yet present. This
+subsection provides several recommendations to better integrate this frontend
+into Eclipse and Spoofax Eclipse.
+
+\subsubsection{Building languages and projects}
+
+TODO: write this when the discussion on classpath issues has been written.
+Mention that there should be a button to build the language/project, put it on
+the classpath and launch it in a REPL.
```

Comment:
I can do this, but this won't be a general solution to the problem: the console frontend and future other frontends likely aren't using OSGi.